Example/metro.config.js

```diff
@@ -1,17 +1,31 @@
-/**
- * Metro configuration for React Native
- * https://github.com/facebook/react-native
- *
- * @format
- */
+/* Metro configuration for React Native
+* https://github.com/facebook/react-native
+*
+* @format
+*/
 
-module.exports = {
-  transformer: {
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: true,
-      },
-    }),
-  },
+const { getDefaultConfig, mergeConfig } = require('@react-native/metro-config');
+
+const defaultConfig = getDefaultConfig(__dirname);
+
+const {
+ resolver: { sourceExts, assetExts },
+} = getDefaultConfig(__dirname);
+
+const config = {
```

Comment:
make sure this config is correct

Reply:
```
missing-asset-registry-path could not be found within the project or in these directories:
  node_modules/react-native/node_modules
  node_modules
  ../node_modules
  ../../node_modules
> 1 | �PNG
  2 | 
  3 | 
  4 | IHD��IDATxڝՅ
```

It was added to resolve this. Works fine